### ISSUE TITLE
Map, Backpack, Settings, Team need to be activity

### ISSUE THREAD

**Issue by andr7430** [SyndicateFighter/SyndicateFighter#10]
So after reading through some design guidelines Map, Backpack, Settings, Team all needs to be its own activity. 
Then backpack will have different fragments, ie overview of all items fragment, and detailed view of a single item fragment.
Sorry about the changes, I know in my diagram I set them as fragments.

**Comment by nohe427** [comment]
So should we not work on the backpack class?

**Comment by nohe427** [comment]
Or the team class?

**Comment by andr7430** [comment]
No, this does not affect the ArcMonster Engine Classes (imma refer to it as the "Engine" now) :fire_engine: 

Just affects the android implementation.

**Comment by nohe427** [comment]
After Mohini sends her next pull request, I will change the repo name to match what it is.  might change package information too to match our current convention.
